sound/soc/sof/ipc3-topology.c

```diff
@@ -1669,6 +1669,12 @@ static int sof_ipc3_control_load_bytes(struct snd_sof_dev *sdev, struct snd_sof_
 	scontrol->ipc_control_data = kzalloc(scontrol->max_size, GFP_KERNEL);
 	if (!scontrol->ipc_control_data)
 		return -ENOMEM;
+	scontrol->old_ipc_control_data = kzalloc(scontrol->max_size, GFP_KERNEL);
```

Comment:
Since we suppose to copy to here, `kmalloc` should be OK, no need to zero it.

I still think that this is better to do dynamically whenever a bytes_ext is put, the other kcontrols are not using this and we just increase or memory footprint for not reason.

Reply:
@paulstelian97 lets go with Peter's suggestion.